12.11_k번째_수.js

```diff
@@ -0,0 +1,11 @@
+function solution(array, commands) {
+    let arr2 =[]
+        for(let num1 = 0; num1 <= commands.length-1; num1++){
```

Comment:
```suggestion
        for(let i= 0; i<= commands.length-1; i++){
```
```for문```을 사용할 땐 ```i, j ,k```과 같은 단어들을 주로 사용합니다. ```i```가 ```index```의 약자로도 쓰일 수 있기 때문에 그런듯?